typos.toml

```diff
@@ -0,0 +1,8 @@
+# See https://github.com/crate-ci/typos/blob/master/docs/reference.md to configure typos
+[default.extend-words]
+ba = "ba"
+fo = "fo"
+enew = "enew"
+tabe = "tabe"
+[files]
+extend-exclude = ["CHANGELOG.md"]
```

Comment:
Thanks, can you explain why you added this line?

Reply:
Ignore this file.
I think it's better to keep it the same as PRs' titles, even if there're some typos.

Reply:
The file does not include the titles of the pull requests. I edit the file manually and summarize the important changes.

Reply:
I see. I thought it was automatically generated. Let me include it in the scope of checking.

Reply:
Do you prefer a commit in a PR or multiple commits?